engines/grim/gfx_opengl_shaders.cpp

```diff
@@ -2004,8 +2017,8 @@ void GfxOpenGLS::prepareMovieFrame(Graphics::Surface* frame) {
 		glGenTextures(1, &_smushTexId);
 	}
 	glBindTexture(GL_TEXTURE_2D, _smushTexId);
-	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
-	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
```

Comment:
this is part of movie playback, please revert change

Reply:
Do you want to revert https://github.com/tobiaspfaff/grim_mouse/commit/8a6806afb71c182f4f2d28f8e9e9a923b5704a00 entirely?

Reply:
yep